packages/dev/core/src/Meshes/greasedLineMesh.ts

```diff
@@ -244,7 +246,10 @@ export class GreasedLineMesh extends Mesh {
     /**
      * Gets the pluginMaterial associated with line
      */
-    get greasedLineMaterial() {
+    get greasedLineMaterial(): IGreasedLineMaterial {

```

Comment:
at least according to the typescript return statement, this could be undefined when returned. Is that the case?

Reply:
Code changed. Now it doesn't cast undefined to `GreasedLinePluginMaterial` and it's clear that it can return `undefined`. Shouldn't I return `null` instead of `undefined` ?

```javascript
    get greasedLineMaterial(): IGreasedLineMaterial | undefined {
        if (this.material && this.material instanceof GreasedLineSimpleMaterial) {
            return this.material;
        }
        const materialPlugin = this.material?.pluginManager?.getPlugin(GreasedLinePluginMaterial.GREASED_LINE_MATERIAL_NAME);
        if (materialPlugin) {
            return <GreasedLinePluginMaterial>materialPlugin;
        }
        return;
    }
```

Reply:
nope, this is perfectly fine